ci-operator/step-registry/telcov10n/functional/cnf-network/cnf-config/telcov10n-functional-cnf-network-cnf-config-commands.sh

```diff
@@ -25,5 +25,5 @@ export CLUSTER_NAME=${CLUSTER_NAME}
 echo CLUSTER_NAME=${CLUSTER_NAME}
 
 cd /eco-ci-cd/
-ansible-playbook ./playbooks/cnf/deploy-cnf-config.yaml -i ./inventories/ocp-deployment/deploy-ocp-hybrid-multinode.yml \
+ansible-playbook ./playbooks/cnf/deploy-cnf-config.yaml -i ./inventories/ocp-deployment/build-inventory.py \
```

Comment:
You need to provide the path to host_vars dir to the build-inventory.py

Same for other commands using the same method.